frontend/src/components/auth/login/login.js

```diff
@@ -175,4 +192,30 @@ const Login = () => {
   );
 };
 
+// const togglePassword = document.querySelector('#togglePassword');
```

Comment:
Remove this whole commente dsection from,[here](https://github.com/HITK-TECH-Community/Community-Website/pull/188/files#diff-d5c81cb0d6ea549df508bdd0e919acef604720ffcee342acea1c1170ed52f506R195) to [here](https://github.com/HITK-TECH-Community/Community-Website/pull/188/files#diff-d5c81cb0d6ea549df508bdd0e919acef604720ffcee342acea1c1170ed52f506R220)